### PR TITLE
Add info about release notifications

### PR DESCRIPTION
**Pull request by laviniat1996**
Thought this might be something nice to add. There was a user asking about this feature today on Discord.

### CODE REVIEW

**Review by meaghanfitzgerald — APPROVED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 1]
```diff
@@ -6,6 +6,20 @@
 
 :::
 
+:::info
+
+If you have an ID for an node, you can use the
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:11]:
```suggestion
If you have a node, you can subscribe to the
```

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 2]
```diff
@@ -6,6 +6,20 @@
 
 :::
 
+:::info
+
+If you have an ID for an node, you can use the 
+[avalanche notify service](/docs/nodes/maintain/avalanche-notify.md)
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:12]:
```suggestion
[avalanche notify service](/docs/nodes/maintain/avalanche-notify.md) with your node ID
```

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 3]
```diff
@@ -6,6 +6,20 @@
 
 :::
 
+:::info
+
+If you have an ID for an node, you can use the 
+[avalanche notify service](/docs/nodes/maintain/avalanche-notify.md) 
+to be notified about new releases.
+
+In addition, if you don't have a node ID, you can get release notifications from github.
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:15]:
```suggestion
In addition, or if you don't have a node ID, you can get release notifications from github.
```

Comment (yulin-dong) [reply docs/apis/avalanchego/avalanchego-release-notes.md:15]: Please make the same modification to the other place in this PR.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md (hunk 1 above)
Comment (laviniat1996) [reply docs/apis/avalanchego/avalanchego-release-notes.md]: fixed 415e7fb55a826fe9cbc3dc82d45d1e7ad050bbee

File: docs/apis/avalanchego/avalanchego-release-notes.md (hunk 2 above)
Comment (laviniat1996) [reply docs/apis/avalanchego/avalanchego-release-notes.md]: fixed 415e7fb55a826fe9cbc3dc82d45d1e7ad050bbee

File: docs/apis/avalanchego/avalanchego-release-notes.md (hunk 3 above)
Comment (laviniat1996) [reply docs/apis/avalanchego/avalanchego-release-notes.md]: fixed 415e7fb55a826fe9cbc3dc82d45d1e7ad050bbee

File: docs/nodes/maintain/upgrade-your-avalanchego-node.md [hunk 4]
```diff
@@ -106,6 +106,20 @@ If you want to download a pre-built binary instead of building it yourself, go
 to our [releases page](https://github.com/ava-labs/avalanchego/releases), and
 select the release you want (probably the latest one.)
 
+:::info
+
+If you have a node, you can subscribe to the
+[avalanche notify service](./avalanche-notify.md)
```
Comment (yulin-dong) [inline docs/nodes/maintain/upgrade-your-avalanchego-node.md:112]:
```suggestion
[avalanche notify service](./avalanche-notify.md) with your node ID
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**